tools/status_monitor/README.md

```diff
@@ -0,0 +1,207 @@
+# Local OTA status monitor util
+
+A helper util for querying specific otaclient's status API and showing the OTA status in real time in the terminal window. If multiple ECUs' status are included, all of these ECUs' status(up to 10) will be displayed.
+
+## Usage
+
+This helper util requires otaclient installed. Prepare a virtual environment with otaclient installed, and then execute the package as follow:
+
+```bash
+# get the source code
+$ git clone https://github.com/tier4/ota-client
+...
+
+# prepare virtual environment
+$ python3 -m venv venv
+$ . venv/bin/activate
+(venv) $
+
+# install otaclient
+(venv) $ pip install -e ota-client
```

Comment:
older pip version doesn't support `-e` option w/o setup.py, so I'd like to add `pip install -U pip` before hand.

Reply:
Updated at [d6fe9ac](https://github.com/tier4/ota-client/pull/251/commits/d6fe9acc13f444c6e8fd6e5be924f1ee897b19e8).